variables.tf

```diff
@@ -3,10 +3,22 @@ variable "cf_org_name" {
   type        = string
 }
 
+variable "cf_space_name" {
```

Comment:
The `name_postfix` description also needs to be updated to reflect how it is used based on this change.

Reply:
space creation dropped completely. We should indeed still use `name_postfix` in the app names to prevent clashes and e.g. to allow mulitple Thanos instances in the same space without name collisions. Will look into this.

Reply:
Yeah i think that makes sense to have the option of postfixing them.

Maybe the default should be blank and the postfix optional?